packages/devtools-network-console/src/reducers/websocket/index.ts

```diff
@@ -96,6 +84,19 @@ export default function reduceWebsocket(collection: WS_State = DEFAULT_WS_STATE,
         };
         return collection.set(reqId, state);
     }
+    if (action.type === 'REQUEST_WEBSOCKET_CLEAR_MESSAGES') {
+        const reqId = action.requestId;
+        let state = collection.get(reqId);
+        if (!state) {
+            return collection
+        }
+        state = {
+            ...state,
+            connected: false,
```

Comment:
Is this true? Can you clear messages without disconnecting?

Reply:
(In other words, can you just omit `connected` as a property here altogether?)

Reply:
That's a good point. I'll omit connected as well. matches up better with the action description. As discussed, this is used when a tab updates the request sent to not be a ws request anymore, so we need to clear out the queue. I'll make sure only clearing messages doesn't cause side effects.

Reply:
omitted and still works as expected. Thanks!